articles/logic-apps/move-logic-app-resources.md

```diff
@@ -36,7 +36,7 @@ To move a resource, such as a logic app or integration account, to another Azure
 
 1. In the [Azure portal](https://portal.azure.com), find and select the logic app resource that you want to move.
 
-1. On the resource's **Overview** page, next to **Subscription**, select the **change** link.
+1. On the resource's **Properties** page, under **Subscription Name**, select the **Change subscription** link.
```

Comment:
Actually, on the **Overview** page, I see the following UX, so here's my suggested edits:

![image](https://user-images.githubusercontent.com/13206671/205704681-17ffc536-ddf8-452e-8372-29254032371e.png)


```suggestion
1. On the resource navigation menu, select **Overview**. Next to the **Subscription** label, select **move**.

    You can also go to the resource's **Properties** page, and under **Subscription Name**, select **Change subscription**.
```